src/Chunk.ts

```diff
@@ -133,6 +136,22 @@ export default class Chunk {
 		this.name = makeLegal(id);
 	}
 
+	generateMangledExportNames () {
+		let curIndex = 9;
+		this.mangledExportNameMap = {};
+		Object.keys(this.exports).forEach(exportName => {
+			if (exportName[0] === '*')
+				return;
+			let mangledName: string;
+			do {
+				curIndex++;
+				mangledName = curIndex.toString(36);
+			}
+			while (mangledName.charCodeAt(0) > 47 && mangledName.charCodeAt(0) < 58)
+			this.mangledExportNameMap[exportName] = mangledName;
```

Comment:
I've never seen do/while loop with a curled `do`, but without a curled `while`. At first I mistakenly thought the line above was part of a `while` loop without indenting. Is this a `prettier` style thing?

Reply:
This is the post-checking form of the `while` loop: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Statements/do...while

Reply:
After a few minutes I recognized it was a do/while loop - not a while loop with line 151 as the body. My point is that the final `while` should be curled as follows with a trailing semicolon to prevent confusion and match common code formatting convention:
```diff
+			} while (mangledName.charCodeAt(0) > 47 && mangledName.charCodeAt(0) < 58);
+			this.mangledExportNameMap[exportName] = mangledName; 
```
Should `prettier` be run on the code base it would do something similar.